src/main/java/pl/mateusz/zadanie_20/HelloController.java

```diff
@@ -0,0 +1,48 @@
+package pl.mateusz.zadanie_20;
+
```

Comment:
tutaj checkstyle pokazuje mi błąd, czy chodzi o "_" nazwie package?

Reply:
Tak, to problem z podkreśleniem